docs: add guarantee note for resume-xmlupload (#868)
dasch-swiss/dsp-tools: docs/cli-commands.md

## `resume-xmlupload`

| <center>Warning</center>                                                    |
|--------------------------------------------------------------------------|

> | There is no warranty that the state of an xmlupload is cleanly saved after `Ctrl + C`. We only guarantee this for `dsp-tools xmlupload --interrupt-after`. |

Author: ```suggestion
| We do not guarantee that the state of an xmlupload is cleanly saved after `Ctrl + C`. We only guarantee this for `dsp-tools xmlupload --interrupt-after`. |
```
